service/history/timerQueueProcessorBase_test.go

```diff
@@ -66,7 +66,8 @@ type (
 		notificationChan chan struct{}
```

Comment:
FYI timerQueueProcessor_test.go and timerQueueProcessor2_test.go needs rewrite (to become a unit test, not integration test which includes db)

Reply:
yes, I saw that.. will change as part of future diffs..